install.sh

```diff
@@ -19,16 +20,19 @@ else
   PLATFORM="linux"
 fi
 
-
-TARNAME=gql-lint-$VERSION-$PLATFORM-$ARCH.tar.gz
+# Stay backwards compatible with older versions that had the version in the file name.
+if [[ "v1v2v3" == *"$VERSION"* ]]; then
+  TARNAME=gql-lint-$VERSION-$PLATFORM-$ARCH.tar.gz
+else
+  TARNAME=gql-lint-$PLATFORM-$ARCH.tar.gz
+fi
+URL="https://github.com/sketch-hq/gql-lint/releases/download/$VERSION/$TARNAME"
```

Comment:
Ummm, I think we are overcomplicating things. We are the only owners and users of gql-lint now. I think it will be fair to nuke all old tags and create a new v0 that's the current version and make a v1 at the end of the cycle. WDYT?

Reply:
Didn't consider that approach as I didn't know you could delete a release 😅 But we already have v1/v2 in "production". Wouldn't nuking all releases break all the GH flows? Can't we we simply delete this snippet of code once the GH actions are updating to use the latest release? We need to update the GH actions anyways to specify `latest` as the version...

Reply:
I buy it. Maybe add an issue to the board linking to this and I will see if I can do some clean up in cooldown